AfriSplash-Remotely/Afrisplash-Frontend
# Contact modal

### Background 
 A Pop-up Modal was needed for the Contact Us button on the Landing page and the Talk to Us button on the Hire Talent page  

### Fixes 
[Link to the issue](https://github.com/AfriSplash-Remotely/Afrisplash-Frontend/issues/136)

### Change Summary
Added a Contact Us Pop Up Modal 

### Screenshots of the change
![Screenshot 2023-08-23 133846](https://github.com/AfriSplash-Remotely/Afrisplash-Frontend/assets/86838623/a114dba0-88e3-43bd-b866-05e780c273c0)


### Additional Notes
None

### Testing Procedure
Click on the Contact Us button on the landing page and the Talk to Us button on the Hire Talent page to see the Contact Modal

### Related PRs or Issues 
None

#### PR Checklist
